generators/server/index.js

```diff
@@ -706,16 +716,6 @@ module.exports = JhipsterServerGenerator.extend({
             }
         },
 
-        writeServeri18nFiles: function () {
-
-            // install all files related to i18n if translation is enabled
-            if (this.enableTranslation) {
-                this.installI18nResFilesByLanguage(this, SERVER_MAIN_RES_DIR, 'en');
-                this.installI18nResFilesByLanguage(this, SERVER_MAIN_RES_DIR, 'fr');
-            }
-            this.template(SERVER_MAIN_RES_DIR + 'i18n/_messages_en.properties', SERVER_MAIN_RES_DIR + 'i18n/messages.properties', this, {});
```

Comment:
Aha okey sorry I missed that. You are correct. Then I'll have to rethink this a bit. I was not sure how these subgenerarors was used. There's to many new things for me here but now I get it. 

Okey so this is what I will do. I will prompt for language in bother server an client generator and set the result in commonOptions. If the result already exists there will be no promt and the commonOptions will be used. This will solve the issue you are talking about. 

Then also I'm thinking of moving the languages subgen call from the app generator into the client and server subgenerator and then call it with a skip client and skip server flags. 
That may make more sense.